src/sqlmodel.cpp

```diff
@@ -122,7 +122,7 @@ Row<Table> SqlModel::at(const int &i) const
 
 SqlModelPrivate::SqlModelPrivate(SqlModel *parent)
 {
-
+    Q_UNUSED(parent);
```

Comment:
The trailing ; is not necessary here.

Reply:
> The trailing ; is not necessary here.

ok

Reply:
Hey guys,
I do not agree with keeping this class in the current situation. I think the structure of this class needs to change. I have concerns about this, but I don't have enough time to do it